src/js/tech/tech.js

```diff
@@ -322,6 +322,7 @@ class Tech extends Component {
     }
 
     let textTracksChanges = Fn.bind(this, function() {
+      let tracks = this.textTracks();
```

Comment:
Is this necessary? Seems like this function could not run unless `tracks` was already defined at the assignment in `emulateTextTracks` above.